### PR TITLE
Added tag for simulated waveform data in characterize_start_end workf…

### PR DESCRIPTION
**Pull request by llondon6**
…low, and changed the padding amount in lengthalign

### CODE REVIEW

File: nrutils/core/nrsc.py [hunk 1]
```diff
@@ -3852,7 +3855,7 @@ def lengthalign(this,yref,apply=True,kind=None):
             l1 = that.hlm[0].n
             l2 = zref.hlm[0].n
 
-        if l1!=l2: ( that if l2>l1 else zref ).pad( max([l1,l2]) )
+        if l1!=l2: ( that if l2>l1 else zref ).pad( abs(l2 - l1) ) #changed to difference rather than max
```
Comment (llondon6) [inline nrutils/core/nrsc.py]: Has this been shown to work with non-spimulated waveforms?

File: nrutils/core/nrsc.py [hunk 2]
```diff
@@ -2959,7 +2959,9 @@ def calchlm(this,w22=None):
         # NOTE that this is the end of the calchlm method
 
     # Characterise the start of the waveform using the l=m=2 psi4 multipole
-    def characterize_start_end(this,turnon_width_in_cylcles=3):
+    def characterize_start_end(this,turnon_width_in_cylcles=3, simulated = False):
```
Comment (llondon6) [inline nrutils/core/nrsc.py]: What does simulated mean in this context? (Isn't it all simulated?)

**Comment by llondon6** [comment]
Hi Jonathan, I've aded a few questions before merging. Thanks for the code changes!

File: nrutils/core/nrsc.py (hunk 1 above)
Comment (thompsonphys) [reply nrutils/core/nrsc.py]: This should work for any input waveform and doesn't depend on the (non-)existence of junk radiation, though I haven't checked the implementation on real waveforms. 

My reason for the change is this: the default behavior of "pad" is to extend a particular array by the argument of pad, in which case padding by the maximum length adds the length of the longer array to the length of the shorter.

E.g. given two arrays, one of length 200 and one of length 300, this function would extend the shorter array by 300, making an array of length 500.

Assuming we want the arrays to be the same length, giving the difference between the two should extend the shorter array to be equal in length to the longer array.

File: nrutils/core/nrsc.py (hunk 2 above)
Comment (thompsonphys) [reply nrutils/core/nrsc.py]: This is a good point. Perhaps a better tag would be "no_junk" or "clean_start"?

**Pull request merged**